Fog.js

```diff
@@ -868,7 +868,11 @@ function setup_draw_buttons() {
 
 
 		var target = $("#fog_overlay");
-		target.css("z-index", "50");
+		if (!e.currentTarget.id || (e.currentTarget.id !== "measure-button" && e.currentTarget.id !== "select-button")) {
+			target.css("z-index", "50");
+		} else {
+			target.css("z-index", "20");
+		}

```

Comment:
What are you trying to do here? 🤔 
Can you run me through the logic, please?

Reply:
yes, to be able to draw over tokens, I set z-indexes to bring the drawing layer on top of the tokens layer.
for the measure and select this doesn't apply as these tools interact directly with the tokens.

Reply:
I think it's better that we definitely change the behaviour for "measure" to stop it interacting with tokens.
So this "exception" for change of z-index should only be for "select".

I think in the future we should create some ui to display token movement directly inside the draggable options of the token.. so everytime you move a token you alway get that measurament without having to go to the measure tool.

Reply:
cool, going to make it only for the select then!

Reply:
done